### ISSUE TITLE
[BUG] Exceptions when loading races from the year 1990

### ISSUE THREAD

**Issue by JoelvanIngen** [theOehrly/Fast-F1#400]
### Describe the issue:

Loading races from the year 1990 seems to fail. I have tried about 8 different event numbers, and none of them loaded properly. The issue arises when using the load function on a get_session. Loading races after the year 2000 does seem to work.

### Reproduce the code example:

```python
import os
import fastf1 as ff1

if not os.path.exists('test_cache'):
    os.mkdir('test_cache')
ff1.Cache.enable_cache('test_cache')

race = ff1.get_session(1990, 10, 'R')
race.load(laps=False, telemetry=False, weather=False, messages=False)
```

### Error message:

```shell
C:\Users\jcing\anaconda3\envs\fastf1\python.exe D:\SynologyDrive\Python\f1-teammate-ranking\test.py
core           INFO 	Loading data for Hungarian Grand Prix - Race [v3.0.4]
logger      WARNING 	Failed to load results data!
Traceback (most recent call last):
  File "D:\SynologyDrive\Python\f1-teammate-ranking\test.py", line 10, in <module>
    race.load(laps=False, telemetry=False, weather=False, messages=False)
  File "C:\Users\jcing\anaconda3\envs\fastf1\Lib\site-packages\fastf1\core.py", line 1245, in load
    _logger.info(f"Finished loading data for {len(self.drivers)} "
                                                  ^^^^^^^^^^^^
  File "C:\Users\jcing\anaconda3\envs\fastf1\Lib\site-packages\fastf1\core.py", line 1058, in drivers
    return list(self.results['DriverNumber'].unique())
                ^^^^^^^^^^^^
  File "C:\Users\jcing\anaconda3\envs\fastf1\Lib\site-packages\fastf1\core.py", line 1066, in results
    return self._get_property_warn_not_loaded('_results')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\jcing\anaconda3\envs\fastf1\Lib\site-packages\fastf1\core.py", line 1047, in _get_property_warn_not_loaded
    raise DataNotLoadedError("The data you are trying to access has not "
fastf1.core.DataNotLoadedError: The data you are trying to access has not been loaded yet. See `Session.load`

Process finished with exit code 1
```

**Comment by theOehrly** [comment]
Ergast apparently provides less information for older sessions. For example the driver abbreviation (called 'driverCode' in Ergast) is unavailable.

This is a simple problem to fix. I'll try to make a patch release within the next day.

**Comment by theOehrly** [comment]
This is fixed in v3.0.6 now
